Buildings/Applications/DHC/EnergyTransferStations/CoolingIndirect.mo

```diff
@@ -1,9 +1,10 @@
 within Buildings.Applications.DHC.EnergyTransferStations;
 model CoolingIndirect
   "Indirect cooling energy transfer station for district energy systems"
-  extends Buildings.Fluid.Interfaces.PartialFourPort(
+  extends Buildings.Fluid.Interfaces.PartialFourPortInterface(
     redeclare final package Medium1 = Medium,
-    redeclare final package Medium2 = Medium);
+    redeclare final package Medium2 = Medium,
+    show_T = true);
```

Comment:
add
`final m1_flow_nominal = mDis_flow_nominal,
  final m2_flow_nomnal = mBui_flow_nominal,
`
and change the name of below parameter `m1_flow_nominal` to `mDis_flow_nominal`, `m2_flow_nominal` to `mBui_flow_nominal`.
The reason for the change is that current setting would trigger warning since the attribute `start = 0.5` is not in the `m1_flow_nominal` of class `Buildings.Fluid.Interfaces.PartialFourPortInterface`.